src/locales/en.json

```diff
@@ -1,9 +1,10 @@
 {
   "_display_name": "English",
-  "about": {
-    "description": "The {{applicationName}} app is made available by {{healthAuthorityName}}",
+  "connect": {
+    "description": "The {{applicationName}} app is made available by {{healthAuthorityName}}.\n\nFor more information, visit the ",
```

Comment:
I think that this text should not change if this is meant to be the same as the About screen text. This text will be replaced by configuration from each of the authorities so maybe we should leave the default as is. Probably should confirm with product.

Reply:
We should rollback all of these language file changes and use the exact same text, as we dont have time to wait for the translations. We can updated it in a future commit.

Reply:
I see, thanks that's helpful. I'm going to merge for now just to get this through but will update in the next pass!